common/main/kgp.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8" ?>
```

Comment:
Looks good, except that we put locales in /seed/ until they get fleshed out.

Reply:
Noted. I've updated the PR. Tks, Mark.